k8s/values.yaml

```diff
@@ -0,0 +1,16 @@
+replicaCount: 1
+
+image:
+  repository: pennlabs/example-django
```

Comment:
These need to be changed, right? We should add documentation for that.

Reply:
yup. if we script this out, i'd like to integrate this into the script.

Reply:
We should definitely look into django project templates, and then hook all of that into an init script.

Reply:
Since this is being pushed to a PR at a later date. Marking as resolved.